validation/validate.py

```diff
@@ -1,35 +1,53 @@
 #!/usr/bin/env python
 
+import sys
+
 import click
 import yaml
 import glob
 
+from pydantic import ValidationError
+
 from schema import CollectionSchema
 
+
 @click.command()
 @click.argument('collection_path', nargs=1)
 def validate(collection_path):
-
     collections = glob.glob(f"{collection_path}/*.yml")
+    yaml_error_list = []
+    schema_error_list = []
 
     for collection in collections:
-
-        # load the Collection
         with open(collection, "r") as stream:
             try:
                 in_data = yaml.safe_load(stream)
-            except yaml.YAMLError as exc:
-                print(exc)
-
-        # validate it
-        # try:
-        #     CollectionSchema().load(in_data)
-        # except ValidationError as err:
-        #     pprint(err.messages)
-
-        print(f"validating {collection}")
-        parsed = CollectionSchema.parse_obj(in_data)
-        print(parsed)
+                CollectionSchema.parse_obj(in_data)
+            except yaml.YAMLError as err:
+                yaml_error_list.append(f'{err.problem} {err.problem_mark}')
+                continue
```

Comment:
Do we need this `continue`? If we catch this exception, wouldn't the workflow immediately move on to the next iteration after appending to `yaml_error_list`?